include/deal.II/dofs/dof_handler.h

```diff
@@ -1224,6 +1241,31 @@ DoFHandler<dim,spacedim>::get_fe () const
 
 template <int dim, int spacedim>
 inline
+const FiniteElement<dim,spacedim> &
+DoFHandler<dim,spacedim>::get_finite_element
+(const unsigned int) const
```

Comment:
Hmm... I find it convenient to write `dof_handler.get_finite_element()` instead of `dof_handler.get_finite_element(0)` in case you are dealing with a `DoFHandler` and not a `hp::DoFHandler`.